internal/provider/slo_data_source_test.go

```diff
@@ -59,3 +60,86 @@ data "honeycombio_slo" "test" {
 		},
 	})
 }
+
+func TestAcc_MDSLODataSource(t *testing.T) {
+	ctx := context.Background()
+	c := testAccClient(t)
+
+	if c.IsClassic(ctx) {
+		t.Skip("Classic does not support multi-dataset SLOs")
+	}
+
+	dataset_all := "__all__"
+	dataset1, err := c.Datasets.Create(ctx, &client.Dataset{
```

Comment:
nit: could we prefix all of these test datasets, DCs, and SLOs with `test.`? There's a helper to do it (e.g. `test.RandomStringWithPrefix("test.", 10)`). It'll make automated cleanup far easier going forward